WordPress/src/main/java/org/wordpress/android/ui/ActionableEmptyView.kt

```diff
@@ -145,5 +152,7 @@ class ActionableEmptyView : LinearLayout {
         }
 
         announceForAccessibility("${title.text}.$subTitle")
+
+        subtitle.contentDescription = null
```

Comment:
This feels like an odd place to have a `contentDescription` being set to `null`. I wouldn't expect a method like `announceEmptyStateForAccessibility` to do that. 

I investigated a bit and it looks like the cause of the issue is `ReaderPostListFragment` is being reused whenever we switched from _Saved Posts_ to _Search_ and to other filters. This means that the `mActionableEmptyView` is behaving as it should but it's **state** is just not being reset properly. There's a `contentDescription` not being reset somewhere. 

The fix should most probably be in how `ReaderPostListFragment` uses `ActionableEmptyView`. What do you think?

Reply:
That's a good point especially since that's the location where this issue is arising. Will address. Thanks for looking into this 🙏

Reply:
I made some changes 465485e Let me know if this is along the lines that you are thinking.

Reply:
Looks okay but probably a tad unnecessary considering that `ReaderPostListFragment` uses `mActionableEmptyView.subtitle` directly. It's probably simpler to just do this instead of creating a function:

```
mActionableEmtpyView.subtitle.setContentDescription(null);
```

If we do want functions, a better, more elaborate, but promotes encapsulation is to have:

```
class ActionableEmptyView {
    private lateinit var subtitle: WPTextView // <-- private

   func setSubtitleText(text: String) {
      subtitle.text = ...
      subtitle.contentDescription = ...
   } 
}
```

Hiding `subtitle` and having `setSubtitleText` means we'll never forget to set `contentDescription` again. This is just a rough translation of what I'm thinking. The work would definitely be much bigger. 😄 

Anyway, what do you think of simplifying it?

Reply:
Good points. 

> mActionableEmtpyView.subtitle.setContentDescription(null);

Yes, I was thinking to do this but didn't want to access it directly but it's already public so might as well access it directly. 

> Anyway, what do you think of simplifying it for now?

I will just use the first option for now since it's highly unlikely this will popup again because most `TextView` objects don't need a content description since the text acts like that.

Reply:
Simplified  🙇 5834fd1